tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -5,67 +5,6 @@
 from tests.metaworld.envs.mujoco.sawyer_xyz.utils import check_success
 
 
-test_cases = [
```

Comment:
is this intentional?

Reply:
Yes. It was removed a few commits ago, but got added back by somebody when they rebased. This list gets overwritten later in the file by all of the for loops.